src/Sylius/Bundle/ApiBundle/Exception/OrderCancellationFailedException.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Exception;
+
+final class OrderCancellationFailedException extends \RuntimeException
```

Comment:
Maybe create a semi-generic exception, smth like `TransitionFailedException` and have all of those extend that, we'll then have one line in status code mapping, and if someone wants to change the status code in an end-app they can do it either for all children at once, or for a specific one.